src/contact_form/contact_form.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import './contact_form.css'
+
+const contacts = [
+    {
+        path: 'tag',
+        title: 'Address',
+        text: '236 5th SE Avenue, New York NY10000, United States',
+        seconLineText: ''
+    },
+    {
+        path: 'bxs_phone',
+        title: 'Phone',
+        text: 'Mobile: +(84) 546-6789 Hotline: +(84) 456-6789',
+        seconLineText: ''
+    },
+    {
+        path: 'clock',
+        title: 'Working Time',
+        text: `Monday-Friday: 9:00 - 22:00 `,
+        seconLineText: 'Saturday-Sunday: 9:00 - 21:00'
```

Comment:
```suggestion
        secondLineText: 'Saturday-Sunday: 9:00 - 21:00'
```
I recommend not to name properties according to their position but semantically. what if this text is rendered not on the second line but on the side? then it loses all meaning. for example this can be named `description`, `additionalInfo` etc.